utHelper.py

```diff
@@ -0,0 +1,372 @@
+            device (str): The device to execute the command on (default: "dut").
+
+        Returns:
+            str: The output/response from the command execution, excluding the command itself and the prompt.
+                    If no response is received, an empty string is returned.
+        """
+        self.log.debug("writeCommand()")
+
+        # Send the command and a newline to the device session
+        self.writeMessageToDeviceSession(command, device=device)
+        self.writeMessageToDeviceSession("\n", device=device)
+
+        # If no prompt is not provided, use the default prompt from the device configuration
+        if prompt is None:
+            prompt = self.getCPEFieldValue("prompt")
+
+        # Wait for the expected prompt or timeout
+        result = self.waitForSessionMessage(prompt, device=device)
+
+        # Flush any additional data from the session
+        result += self.flushSessionData(device=device)
+
+        # Split the result into lines
+        message = result.split("\r\n")
+
+        # If there are 2 or fewer lines, it means there was no command output
+        if len(message) <= 2:
+            return ""
+
+        # Remove the first line (the command itself) and the last line (the prompt)
+        message.pop(0)
+        message.pop(len(message) - 1)
+
+        # Join the remaining lines and return them as the command output
+        return "\r\n".join(message)
+
+## Useful utility functions
+
+    def isStringInList(self, string, searchList):
+        """
+        Checks if a given string is present in a list of strings.
+
+        Args:
+            string (str): The string to search for.
+            searchList (list): The list of strings to search in.
+
+        Returns:
+            bool: True if the string is found in the list, False otherwise.
+        """
+        self.log.step("findStringInList(string:'{}')".format(string))
+
+        # Iterate through the search list and check if the string is present in any of the items
+        for line in searchList:
+            if string in line:
+                return True
+        return False
+
+    ## Useful log cat / save functions for the device
+
+    def catFileOnDevice(self, filePath, device="dut"):
+        """
+        Reads and retrieves the contents of a file on the device using the 'cat' command.
+
+        Args:
+            filePath (str): The path to the file on the device.
+            device (str): The device to read the file from (default: "dut").
+
+        Returns:
+            str or False: The contents of the file as a string if successful, 
+                            False if the session is not open.
+        """
+        self.log.step("catFileOnDevice('{}')".format(filePath))
+
+        # Check if the session is open
+        if not self.session.sessionOpen:
+            self.step.log("catFileOnDevice(): session not open")
+            return False
```

Comment:
Is this something you'd really want in the tests? Wouldn't we ideally copy the file back to the tests' workspace on the host and have the python test read it from there? That way we'd have a copy of the file for debugging when the test fails.